package.json

```diff
@@ -122,8 +122,6 @@
     "react-autosuggest": "9.4.3",
     "react-dom": "16.14.0",
     "react-error-boundary": "3.1.3",
-    "react-instantsearch-dom": "6.12.0",
-    "react-instantsearch-dom-maps": "6.12.0",
```

Comment:
Why this change?

Reply:
they didn't seem needed, but it caused the stories to import from node_modules instead of the local packages @francoischalifour